assets/js/membraneWebRTC.ts

```diff
@@ -160,6 +164,9 @@ export interface Callbacks {
   onConnectionError?: (message: string) => void;
 
   /**
+   * Currently, this callback is only invoked when DisplayManager in RTC Engine is
+   * enabled and simulcast is disabled.
+   *
    * Called when priority of video tracks have changed.
```

Comment:
I think I would swap order here
```suggestion
   * Called when priority of video tracks have changed.
   *
   * This callback is only invoked when DisplayManager in RTC Engine is
   * enabled and simulcast is disabled.
```